README.md

```diff
@@ -154,7 +154,8 @@ map: {
 
 | Output  | Description |
 | ------------- | ------------- |
-| (add)  | Fired when item is selected |
+| (add)  | Fired when item is selected before ngModel updates |
+| (afterAdd)  | Fired after item is selected and ngModel updates |
```

Comment:
I don't think there is a need for separate event, just need to place current `add` correctly

Reply:
alright.